### PR TITLE
update eslint vscode settings

### PR DESCRIPTION
**Pull request by o-az**
small tweaks

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -2,4 +2,6 @@ pnpm-lock.yaml
 LICENSE
 **/generated
 **/.next
-**/dist
\ No newline at end of file
+**/dist
+.gitignore
```
Comment (0xOlias) [inline .prettierignore:6]: Noticed this annoyance locally also

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,13 +1,23 @@
 {
-  "editor.defaultFormatter": "rvest.vs-code-prettier-eslint",
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
```
Comment (0xOlias) [inline .vscode/settings.json:2]: Did you notice any differences after switching this?

**Review by 0xOlias — APPROVED** [review]
LGTM

File: .vscode/settings.json (hunk 2 above)
Comment (o-az) [reply .vscode/settings.json:2]: Yeah not having to have that extra so I was able to uninstall it and keep prettier&eslint exts

**Pull request merged**